### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/dynamics-365-unified-operations-public/commit/a2cba6ab8261fd48f1eb78d697c2acc886c7ca53#commitcomment-71321833

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/dynamics-365-unified-operations-public/commit/cd1a163c899cb70cbea3624b6f08125cfda12772

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

Fwiw, I'm well aware that this is not a tiny PR. It took a non-trivial amount of effort to produce and I tried to be very careful not to make changes to non-English content (with a couple of exceptions that I will tag). I also tried to make sure I renamed files when I changed references (not all files corresponded to references). It will take me a bit of time to annotate this PR. I'm quite happy to split things out as requested.

### CODE REVIEW

**Comment by buck1ey** [comment]
### :white_check_mark: Organization membership check passed
_Requester is not a member of the MicrosoftDocs organization._

File: articles/finance/localizations/apac-sau-allocate-bank-document-charges-shipment.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ Follow these steps to allocate bank document charges.
 5. On the **List** tab, set the **Account** and **Debit** fields.
 6. On the **Letter of credit/import collection** page, on the Action Pane, select **Bank document** \> **Bank document charge**.
 
-    ![Allocating bank document charges.](media/apac-sau-allocate-bank-docment-charge.PNG)
+    ![Allocating bank document charges.](media/apac-sau-allocate-bank-document-charge.PNG)
```
Comment (jsoref) [inline articles/finance/localizations/apac-sau-allocate-bank-document-charges-shipment.md:69]: [renamed](https://github.com/MicrosoftDocs/dynamics-365-unified-operations-public/pull/3964/files#diff-fdd47d83341ac581fa1a4c5e8f57857eb4a166258c4c0f58ca7772e78e4297fc)

File: articles/finance/localizations/emea-deu-gdpdu-audit-data-export.md [hunk 2]
```diff
@@ -123,7 +123,7 @@ The following tables show the Tax data structure definitions.
 |-----|------------------------|-----------|---------------------------------------------|--------------------------------------|
 | 1   | STEUERART              | Zeichen   | Beschreibung der Steuerart                  | $TaxTrans/taxName()                                      |
 | 2   | STEUERBUCHUNGSREFERENZ | Numerisch | Gibt es hierzu eine MWST-Buchung? - lfd Nr. | $TaxTrans/$TaxTransGeneralJournalAccountEntry/$GeneralJournalAccountEntryRecId                                          |
-| 3   | MWST\_CODE             | Zeichen   | MWST Bezeichung                             | $TaxTrans/TaxCode                                        |
+| 3   | MWST\_CODE             | Zeichen   | MWST Bezeichnung                            | $TaxTrans/TaxCode                                        |
```
Comment (jsoref) [inline articles/finance/localizations/emea-deu-gdpdu-audit-data-export.md:126]: This is a German correction...
https://www.google.com/search?q=%22MWST+Bezeichung%22

> Showing results for ["MWST Bezeichnung"](https://www.google.com/search?sxsrf=APq-WBvrEzAh4orDyfaiyCr13LkxIpvxrw:1649991690685&q=%22MWST+Bezeichnung%22&spell=1&sa=X&ved=2ahUKEwiP6OuEipX3AhX4hHIEHcRVBxUQkeECKAB6BAgBEDI)
> Search instead for ["MWST Bezeichung"](https://www.google.com/search?sxsrf=APq-WBvrEzAh4orDyfaiyCr13LkxIpvxrw:1649991690685&q=%22MWST+Bezeichung%22&nfpr=1&sa=X&ved=2ahUKEwiP6OuEipX3AhX4hHIEHcRVBxUQvgUoAXoECAEQMw)

File: articles/finance/localizations/emea-esp-sii.md [hunk 3]
```diff
@@ -127,7 +127,7 @@ The electronic message processing from the previous table works with the followi
...
 | FacturasProveedores               | **Vendor invoices**                                                                                                                                                                                                                                                                                                                                                                                                           | SII              |
-| FacturasСliente                   | **Customer invoices**                                                                                                                                                                                                                                                                                                                                                                                                         | SII              |
```
Comment (jsoref) [inline articles/finance/localizations/emea-esp-sii.md:130]: This is a Cyrillic `S` not a `C`.

File: articles/finance/localizations/emea-esp-sii.md [hunk 4]
```diff
@@ -357,7 +357,7 @@ As a result of this analyzes system defines related **IDType**
 | **Tax registration type** | **IDType in SII system**                                                                |
 |-----------------|--------------------------------------------------------------------------------|
 | `TaxRegistrationTypesList::TAXID`| 02                                                                         |
-| `TaxRegistrationTypesList::Passpor`| 03                                                                       |
+| `TaxRegistrationTypesList::Passport`| 03                                                                       |
```
Comment (jsoref) [inline articles/finance/localizations/emea-esp-sii.md:360]: ?

File: articles/finance/localizations/emea-ita-e-invoices.md [hunk 5]
```diff
@@ -61,7 +61,7 @@ Before you can begin to work with the electronic invoice functionality, the foll
 
 Select the configurations that are used to create electronic invoice XML files for sales and free text invoices, sales and free text credit notes, project invoices, and project credit notes. You can find these configurations on the **Electronic document** tab of the **Accounts receivable parameters** page (**Accounts receivable** \> **Setup** \> **Accounts receivable parameters**).
 
-![Electronic document tab of the Accounts receivable parameters page.](media/emea-ita-electronic-invocies-AR-parameter-e-invoices.png)
+![Electronic document tab of the Accounts receivable parameters page.](media/emea-ita-electronic-invoices-AR-parameter-e-invoices.png)
```
Comment (jsoref) [inline articles/finance/localizations/emea-ita-e-invoices.md:64]: [renamed](https://github.com/MicrosoftDocs/dynamics-365-unified-operations-public/pull/3964/files#diff-035ad379298f2769239b8d08292436af15cc151e8f02949a5bc3a7e6dfc1358d)

File: articles/finance/localizations/emea-ita-e-invoices.md [hunk 6]
```diff
@@ -103,7 +103,7 @@ For the functionality to work, the following data must be set up:
 
 - Electronic document property types (**Accounts receivable** \> **Setup** \> **Electronic document property types**) and the table that each document property type is applicable to. For electronic invoice functionality, the **Customers** and **Legal entities** tables are used.
 
-    ![Setting applicability on the Electronic document property types page.](media/emea-ita-electronic-invocies-electronic-document-property-types.png)
+    ![Setting applicability on the Electronic document property types page.](media/emea-ita-electronic-invoices-electronic-document-property-types.png)
```
Comment (jsoref) [inline articles/finance/localizations/emea-ita-e-invoices.md:106]: [renamed](https://github.com/MicrosoftDocs/dynamics-365-unified-operations-public/pull/3964/files#diff-90326a0189467b8debec371ccae66e33e6939162104e857bd5f77de9ef237dd2)

File: articles/finance/localizations/emea-ita-e-invoices.md [hunk 7]
```diff
@@ -155,15 +155,15 @@ The value of this field is used to define the type of communication (business-to
 
 In both cases, the system enters the value of this field in the **CodiceDestinatario** tag in the XML file.
 
-![Authority office field on the Sales demographics FastTab of a customer record.](media/emea-ita-electronic-invocies-customer-authority-office.png)
+![Authority office field on the Sales demographics FastTab of a customer record.](media/emea-ita-electronic-invoices-customer-authority-office.png)
```
Comment (jsoref) [inline articles/finance/localizations/emea-ita-e-invoices.md:158]: [renamed](https://github.com/MicrosoftDocs/dynamics-365-unified-operations-public/pull/3964/files#diff-19a462413d8e6c8aab2ade20d854d648302279999a66c2afd7b703988ec1f948)

File: articles/finance/localizations/emea-ita-e-invoices.md [hunk 8]
```diff
@@ -155,15 +155,15 @@ The value of this field is used to define the type of communication (business-to
...
 
 #### Activate automatic creation of electronic invoices
 
 Go to **Accounts receivable** \> **Customers** \> **All customers**, and open a customer record in **Edit** mode. Then, on the **Invoice and delivery** FastTab, in the **E-invoice** section, find the **eInvoice register** option. If this option is set to **Yes**, the system automatically creates the record on the **Electronic customer invoices** list page. For more information, see the [Electronic invoice register](#einvoiceregister) section later in this topic.
 
-![E-invoice section on the Invoice and delivery FastTab of a customer record.](media/emea-ita-electronic-invocies-customer-e-invoice.png)
+![E-invoice section on the Invoice and delivery FastTab of a customer record.](media/emea-ita-electronic-invoices-customer-e-invoice.png)
```
Comment (jsoref) [inline articles/finance/localizations/emea-ita-e-invoices.md:166]: [renamed](https://github.com/MicrosoftDocs/dynamics-365-unified-operations-public/pull/3964/files#diff-06b6f3e06d3427294c713f2cea0374e16fe3e63b6c6f2c1f505a2c49c126ea1a)

File: articles/finance/localizations/emea-ita-e-invoices.md [hunk 9]
```diff
@@ -257,7 +257,7 @@ For more information, see the "Invoice types configuration" section in [A countr
 
 To electronically sign electronic invoices by using a certificate of either the **Company** type or the **User** type, go to **Accounts receivable** \> **Setup** \> **Electronic signature certificates**.
 
-![Electronic signature certificates page.](media/emea-ita-electronic-invocies-certificate.png)
+![Electronic signature certificates page.](media/emea-ita-electronic-invoices-certificate.png)
```
Comment (jsoref) [inline articles/finance/localizations/emea-ita-e-invoices.md:260]: [renamed](https://github.com/MicrosoftDocs/dynamics-365-unified-operations-public/pull/3964/files#diff-c40fa12267e17168a06976312ec08543fea50494ba52aabd76578b9ddf155e7c)

File: articles/finance/localizations/emea-ita-e-invoices.md [hunk 10]
```diff
@@ -330,7 +330,7 @@ On the **Electronic customer invoices** page, you can perform the following acti
 > [!NOTE]
 > The **Electronic invoices** page (**Project management and accounting** \> **Project invoices** \> **E-invoices** \> **Electronic invoices**) resembles the **Electronic customer invoices** page and has the same functions.
 
-![Electronic customer invoices page.](media/emea-ita-electronic-invocies-electronic-customer-invoices.png)
+![Electronic customer invoices page.](media/emea-ita-electronic-invoices-electronic-customer-invoices.png)
```
Comment (jsoref) [inline articles/finance/localizations/emea-ita-e-invoices.md:333]: [renamed](https://github.com/MicrosoftDocs/dynamics-365-unified-operations-public/pull/3964/files#diff-efb6ffabe5fa36636c943306d6849c6fce5f9e9503c036c2c24ee25209eef607)

File: articles/finance/localizations/emea-nl-vat-declaration-netherlands.md [hunk 11]
```diff
@@ -630,7 +630,7 @@ To upload the electronic messages data package, follow these steps:
 
 7. Go to **Tax** > **Setup** > **Electronic messages** > **Executable class settings**. Verify the following :
 
-    - The executable class **NLOBGetStatusFromDidgipoort** has the **Electronic reporting import** action type.
+    - The executable class **NLOBGetStatusFromDigipoort** has the **Electronic reporting import** action type.
```
Comment (jsoref) [inline articles/finance/localizations/emea-nl-vat-declaration-netherlands.md:633]: This is notable, but see below

File: articles/finance/localizations/emea-nor-nets-import-format.md [hunk 12]
```diff
@@ -11,7 +11,7 @@ ms.technology:
 
 # optional metadata
 
-ms.search.form: BankCustPaymIdTable, CustEInvoiceIntegrationeInvoice, CustEInvoiceIntegrationTypePaymMode, CustEinvoiceIntegrationTypeTable, CustPaymMode, LedgerJournalTransCustPaym
+ms.search.form: BankCustPaymIdTable, CustEInvoiceIntegrationInvoice, CustEInvoiceIntegrationTypePaymMode, CustEinvoiceIntegrationTypeTable, CustPaymMode, LedgerJournalTransCustPaym
```
Comment (jsoref) [inline articles/finance/localizations/emea-nor-nets-import-format.md:14]: probably

File: articles/finance/localizations/latam-mex-carta-de-porte.md [hunk 13]
```diff
@@ -177,7 +177,7 @@ After you enable the feature, follow these steps to enter additional hazardous m
 > [!NOTE]
 > To select values for the **Regulation code** and **Packing group** fields, first fill in the **Hazardous material regulation** and **Hazardous material packing groups** tables in accordance with the **c\_MaterialPeligroso** and **c\_TipoEmbalaje** SAT catalogs that are located at **Product information management** \> **Setup** \> **Hazardous material shipping documentation**.
 >
-> As of version 10.0.23 (build 10.0.1037.149), you can work with the **Display hazardous status** option to send the **matrialPeligrosso** attribute as output in an XML file. If the **Display hazardous status** option is set to **Yes**, you can set the **Hazardous materials** option to **Yes**.
+> As of version 10.0.23 (build 10.0.1037.149), you can work with the **Display hazardous status** option to send the **materialPeligroso** attribute as output in an XML file. If the **Display hazardous status** option is set to **Yes**, you can set the **Hazardous materials** option to **Yes**.
```
Comment (jsoref) [inline articles/finance/localizations/latam-mex-carta-de-porte.md:180]: Notably not English

File: articles/finance/localizations/norway.md [hunk 14]
```diff
@@ -36,7 +36,7 @@ This topic provides links to documentation resources for Norway.
 - [Nets import format](emea-nor-nets-import-format.md)
 - [VAT statement](emea-nor-sales-tax-payment-report.md)
 - [VAT return with direct submission to Altinn](emea-nor-vat-return.md)
-- [Standard Audit File for Tax (SAF-T)](emea-nor-satndard-audit-file-for-tax.md)
+- [Standard Audit File for Tax (SAF-T)](emea-nor-standard-audit-file-for-tax.md)
```
Comment (jsoref) [inline articles/finance/localizations/norway.md:39]: [renamed](https://github.com/MicrosoftDocs/dynamics-365-unified-operations-public/pull/3964/files#diff-bab056e5180223133e569b468f9956b596e22d88c8de8f07e924cb6c1bedce9c)

File: articles/finance/localizations/rcs-globalization-feature.md [hunk 15]
```diff
@@ -71,7 +71,7 @@ To use a Globalization feature, you must first import it from the the Global rep
 1. Make sure that the **Globalization features** feature is turned on in Feature management, as described earlier in this topic.
 2. Open the new **Globalization Features** workspace, and then, under **Features**, select the **e-Invoicing** tile.
 
-    ![Global Features workspace.](./media/RCS_GlobalF_2%20Feature%20wrkspace.JPG)
+    ![Global Features workspace.](./media/RCS_GlobalF_2%20Feature%20workspace.JPG)
```
Comment (jsoref) [inline articles/finance/localizations/rcs-globalization-feature.md:74]: [renamed](https://github.com/MicrosoftDocs/dynamics-365-unified-operations-public/pull/3964/files#diff-89e2941151c3b5cf06b9fd1087134922a9ac79f03c9820ebd067935a948a4f24)

File: articles/finance/localizations/rus-accounting-reporting.md [hunk 16]
```diff
@@ -81,7 +81,7 @@ ms.dyn365.ops.version: 10.0.1
     1. Go to **Tax \> Setup \> Electronic messages \> Additional fields**.
     2. In the left pane, select the field that is named **EconomicActivityTypeCode (ОКВЭД)**. Then, on the **Value** FastTab, select **Add**. In the **Field value** field, enter the code for legal entity's economic activity type.
     3. Select the field that is named **OrganizationalFormCode (ОКОПФ)**. Then, on the **Value** FastTab, select **Add**. In the **Field value** field, enter the code for the legal entity's organizational form.
-    4. Select the field that is named **OwneshipFormCode (ОКФС)**. Then, on the **Value** FastTab, select **Add**. In **Field value**, enter the code for the legal entity's ownership form.
+    4. Select the field that is named **OwnershipFormCode (ОКФС)**. Then, on the **Value** FastTab, select **Add**. In **Field value**, enter the code for the legal entity's ownership form.
```
Comment (jsoref) [inline articles/finance/localizations/rus-accounting-reporting.md:84]: ?

File: articles/finance/localizations/rus-export-trade.md [hunk 17]
```diff
@@ -455,11 +455,11 @@ The system will reset the current operation type to the previous operation type.
 5. On the **Records to include** FastTab, select **Filter** to specify selection criteria.
 6. Select **OK**.
 
-    ![Outgoing VAT processing - select data for period dialog box.](media/9_Outgiong_VAT_processing_-_select_data_for_period.png)
+    ![Outgoing VAT processing - select data for period dialog box.](media/9_Outgoing_VAT_processing_-_select_data_for_period.png)
```
Comment (jsoref) [inline articles/finance/localizations/rus-export-trade.md:458]: [renamed](https://github.com/MicrosoftDocs/dynamics-365-unified-operations-public/pull/3964/files#diff-2127530b2b518dbf9b2e8dab43fd6d60fc3098425827e966e4516f7f1af5c11e)

File: articles/finance/localizations/rus-export-trade.md [hunk 18]
```diff
@@ -455,11 +455,11 @@ The system will reset the current operation type to the previous operation type.
 5. On the **Records to include** FastTab, select **Filter** to specify selection criteria.
 6. Select **OK**.
 
-    ![Outgoing VAT processing - select data for period dialog box.](media/9_Outgiong_VAT_processing_-_select_data_for_period.png)
+    ![Outgoing VAT processing - select data for period dialog box.](media/9_Outgoing_VAT_processing_-_select_data_for_period.png)
 
 7. On the **Sales book (Outgoing VAT processing)** page, mark factures for posting, and then, on the Action Pane, select **Post**.
 
-    ![Sales book (outgoing VAT processing) page.](media/10_Sales_book_(Outgiong_VAT_processing).png)
+    ![Sales book (outgoing VAT processing) page.](media/10_Sales_book_(Outgoing_VAT_processing).png)
```
Comment (jsoref) [inline articles/finance/localizations/rus-export-trade.md:462]: [renamed](https://github.com/MicrosoftDocs/dynamics-365-unified-operations-public/pull/3964/files#diff-9ce42098aec3a771dab2ab4cdbf05bf479cf46a49cdf0777e590dc326087a38d)

File: articles/finance/localizations/rus-third-party-misc-charges.md [hunk 19]
```diff
@@ -371,7 +371,7 @@ The cost price of an item can be adjusted many times by using the functionality
 ## View the allocation history for miscellaneous charges
 ### View the allocation history from the invoice journal
 
-1. On the **Invoice journa**l page, select purchase invoice that you created earlier that includes the service.
+1. On the **Invoice journal** page, select purchase invoice that you created earlier that includes the service.
```
Comment (jsoref) [inline articles/finance/localizations/rus-third-party-misc-charges.md:374]: drift

File: articles/finance/localizations/tax-engine-trouble-shooting.md [hunk 20]
```diff
@@ -94,7 +94,7 @@ When debug mode is turned on, you can find the selected posting profiles in the
 
 To work correctly, the tax engine relies on input from taxable documents, such as sales and purchase invoices. If you extend the configuration by adding new fields, fields might be incorrectly mapped, or the writing to the data provider might be incorrect. To identify the issue, set the **Check model mapping discrepancies** option to **Yes**. You can view an additional section to show the discrepancies.
 
-![Check model mapping discrepancies option.](media/GTE-model-mapping-deprepancies.png)
+![Check model mapping discrepancies option.](media/GTE-model-mapping-discrepancies.png)
```
Comment (jsoref) [inline articles/finance/localizations/tax-engine-trouble-shooting.md:97]: [renamed](https://github.com/MicrosoftDocs/dynamics-365-unified-operations-public/pull/3964/files#diff-79a22356bbe594b9b5a0276087581cd373a6c5531fd56fe3c97469338401913f)

File: articles/human-resources/hr-admin-integration-ats-api-person-address.md [hunk 21]
```diff
@@ -78,7 +78,7 @@ This entity contains the list of postal addresses for candidate records.
 | **County**<br>mshr_county<br>*String* | Read/write<br>Optional | The county of the address. Set up in mshr_logisticsaddresscountyentity entity. |
 | **Zip Code**<br>mshr_zipcode<br>*String* | Read/write<br>Optional | The zip/postal code of the address. Set up in mshr_logisticsaddresspostalcodeentity entity. |
 | **Country Region ID**<br>mshr_countryregionid<br>*String* | Read/write<br>Optional | The country or region of the address. |
-| **Country/Region ID Value**<br>_mshr_fk_countriregion_id_value<br>*GUID* | Read-only<br>Optional<br>Foreign key: mshr_logisticaddresscountryregionentityid of mshr_logisticsaddresscountryregionentity | System-generated unique identifier of the country/region of the address. |
+| **Country/Region ID Value**<br>_mshr_fk_countryregion_id_value<br>*GUID* | Read-only<br>Optional<br>Foreign key: mshr_logisticaddresscountryregionentityid of mshr_logisticsaddresscountryregionentity | System-generated unique identifier of the country/region of the address. |
```
Comment (jsoref) [inline articles/human-resources/hr-admin-integration-ats-api-person-address.md:81]: ?

File: articles/human-resources/hr-admin-integration-ats-api-recruiting-request-location.md [hunk 22]
```diff
@@ -71,7 +71,7 @@ The list of locations defined as locations where recruited employees will work u
 | **Recruiting Request Location Entity ID**<br>mshr_hcmrecruitingrequestlocationentityid<br>*GUID* | Read-only<br>Required | System-generated unique identifier for the recruiting request location record. |
 | **Description**<br>mshr_description<br>*String* | Read/write<br>Required | Description of the location. |
 | **Country/Region ID**<br>mshr_countryregionid<br>*String* | Read-only<br>Optional | Specifies the country or region where the candidate has citizenship. |
-| **Country/Region ID Value**<br>_mshr_fk_countriregion_id_value<br>*GUID* | Read-only<br>Optional<br>Foreign key: mshr_logisticaddresscountryregionentityid of mshr_logisticsaddresscountryregionentity | System-generated unique identifier of the country/region of the address. |
+| **Country/Region ID Value**<br>_mshr_fk_countryregion_id_value<br>*GUID* | Read-only<br>Optional<br>Foreign key: mshr_logisticaddresscountryregionentityid of mshr_logisticsaddresscountryregionentity | System-generated unique identifier of the country/region of the address. |
```
Comment (jsoref) [inline articles/human-resources/hr-admin-integration-ats-api-recruiting-request-location.md:74]: ?

File: articles/supply-chain/cost-management/bom-calculations.md [hunk 23]
```diff
@@ -84,7 +84,7 @@ The following situations illustrate two uses of the fallback principle:
...
+-   **Order-specific BOM calculation** - A variation of the **BOM calculation** page is used in the context of a sales order, sales quotation, or service order line item. An order-specific BOM calculation doesn't generate a record in the within a costing version. Instead, it generates a calculation record that appears on the **BOM calculation results** page. This calculation record provides the starting point for viewing the calculation details (for example, on the **Calculate item cost** page). Information about a selected calculation record can be transferred to the originating line item. For example, the calculated sales price can be transferred to a sales order line item.
```
Comment (jsoref) [inline articles/supply-chain/cost-management/bom-calculations.md:87]: drift

File: articles/supply-chain/pim/hazmat-overview.md [hunk 24]
```diff
@@ -49,7 +49,7 @@ Product information management provides a range of setup tables where you can en
 The following common regulations were referenced when this functionality was developed:
 
 - **ADR** – Regulations that are related to the international carriage of dangerous goods by road
-- **CFR 49** – Regulations in the United Sates for the carriage of dangerous goods
+- **CFR 49** – Regulations in the United States for the carriage of dangerous goods
```
Comment (jsoref) [inline articles/supply-chain/pim/hazmat-overview.md:52]: Country...

File: articles/supply-chain/pim/product-identifiers.md [hunk 25]
```diff
@@ -157,10 +157,10 @@ When you use the **Released product creation V2** entity to create products, mul
...
+| Manual = No | Manual = Yes | Auto generate | No mapping | Both product numbers and item numbers use the **Item number** number sequence. | Both product numbers and item numbers use the **Product number** number sequence. | Both product numbers and item numbers will follow the product number sequence. This is the recommended approach to import bulk products with the Released product creation V2 data entity.<br><br>You can only use this approach when bulk importing items (several rows) and when you are not creating items through the user interface. If you need both to bulk import and to create products through the user interface, use the procedure in the next row of this table instead. To transition from using a bulk import approach to using the user interface to manually import and create products, you must manually adjust the **Next number** in the item number sequence to match the **Next number** in the product number sequence. Then you could switch to the approach in the next row of this table. |
```
Comment (jsoref) [inline articles/supply-chain/pim/product-identifiers.md:163]:
```diff
-in the item number number sequence
+ in the item number sequence
```

File: articles/supply-chain/toc.yml [hunk 26]
```diff
@@ -3264,7 +3264,7 @@
         - name: Costing parameter values setup
           href: landed-cost/costing-parameters-setup.md
         - name: Default descriptions for the general ledger
-          href: landed-cost/defult-descriptions.md
+          href: landed-cost/default-descriptions.md
```
Comment (jsoref) [inline articles/supply-chain/toc.yml:3267]: [renamed](https://github.com/MicrosoftDocs/dynamics-365-unified-operations-public/pull/3964/files#diff-3f696a8f4553285e24a4deb4d902faf22faa14ed1c59742222de0592ba801726)

File: articles/supply-chain/toc.yml [hunk 27]
```diff
@@ -3710,7 +3710,7 @@
         - name: Directly derived firmed orders are processed by an in-review workflow
           href: troubleshooting/planning/firmed-directly-deri.md
         - name: You can't filter master planning items by their related coverage group values
-          href: troubleshooting/planning/master-planning-batc.md
+          href: troubleshooting/planning/master-planning-batch.md
```
Comment (jsoref) [inline articles/supply-chain/toc.yml:3713]: [renamed](https://github.com/MicrosoftDocs/dynamics-365-unified-operations-public/pull/3964/files#diff-155273da9850f8b915099fa292682396f0ce92b848e39d19f635feacbd023955)

File: articles/supply-chain/toc.yml [hunk 28]
```diff
@@ -3860,7 +3860,7 @@
       - name: Transportation management
         items:
         - name: Can only add the main account as the credit account for reconciliation reasons
-          href: troubleshooting/transportation/reconciliation-reaso.md
+          href: troubleshooting/transportation/reconciliation-reason.md
```
Comment (jsoref) [inline articles/supply-chain/toc.yml:3863]: [renamed](https://github.com/MicrosoftDocs/dynamics-365-unified-operations-public/pull/3964/files#diff-bca784005c799b36c31d941a4477ed34499830c051db50f95d4f57774a072ca4)

File: articles/supply-chain/transportation/reconcile-freight-transportation-management.md [hunk 29]
```diff
@@ -38,7 +38,7 @@ Freight reconciliation can be done manually, or it can be set up to occur automa
 
 Freight rates are calculated by the rate engine that is associated with the relevant shipping carrier. When a load is confirmed, a freight bill is generated, and the freight rates are transferred to it. The freight rates are apportioned as miscellaneous charges to the relevant source document (purchase order, sales order, and/or transfer order), depending on the setup that is used for the regular billing process. The freight reconciliation process (which is also known as the matching process) can start as soon as the freight invoice arrives from the shipping carrier. The invoice can be received electronically or on paper. If the invoice is received on paper, you can generate an electronic invoice by using the freight bill as a template.
 
-[![Freight reconciliation process.](./media/freight-reconcilation-process.jpg)](./media/freight-reconcilation-process.jpg)
+[![Freight reconciliation process.](./media/freight-reconciliation-process.jpg)](./media/freight-reconciliation-process.jpg)
```
Comment (jsoref) [inline articles/supply-chain/transportation/reconcile-freight-transportation-management.md:41]: [renamed](https://github.com/MicrosoftDocs/dynamics-365-unified-operations-public/pull/3964/files#diff-d0710f7025791d6b474187df6a041e371f80d1fcd2627d478247f0bf96e89a31)

File: articles/supply-chain/warehousing/step-icons-titles.md [hunk 30]
```diff
@@ -232,7 +232,7 @@ The following table shows every standard step icon that is currently available,
 <td><img src="media/step-icons-invent-style-id.png" alt="Inventory style ID step icon" title="Inventory style ID step icon"><br>InventStyleID</td>
 <td><img src="media/step-icons-invent-version-id.png" alt="Inventory version ID step icon" title="Inventory version ID step icon"><br>InventVersionID</td>
 <td><img src="media/step-icons-item-id.png" alt="Item ID step icon" title="Item ID step icon"><br>ItemID</td>
-<td><img src="media/step-icons-itm-contianer-id.png" alt="ITM container ID step icon" title="ITM container ID step icon"><br>ITMContainerID</td>
+<td><img src="media/step-icons-itm-container-id.png" alt="ITM container ID step icon" title="ITM container ID step icon"><br>ITMContainerID</td>
```
Comment (jsoref) [inline articles/supply-chain/warehousing/step-icons-titles.md:235]: [renamed](https://github.com/MicrosoftDocs/dynamics-365-unified-operations-public/pull/3964/files#diff-8801091815c58f1f83df7120c4395e60a381d4314462c77450cd19e368b32b9f)

File: articles/supply-chain/warehousing/step-icons-titles.md [hunk 31]
```diff
@@ -303,7 +303,7 @@ The following table shows every standard step icon that is currently available,
 <tr>
 <td><img src="media/step-icons-to-number.png" alt="To number step icon" title="To number step icon"><br>ToNum</td>
 <td><img src="media/step-icons-to-warehouse.png" alt="To warehouse step icon" title="To warehouse step icon"><br>ToWarehouse</td>
-<td><img src="media/step-icons-tranport-load-id.png" alt="Transport load ID step icon" title="Transport load ID step icon"><br>TransportLoadId</td>
+<td><img src="media/step-icons-transport-load-id.png" alt="Transport load ID step icon" title="Transport load ID step icon"><br>TransportLoadId</td>
```
Comment (jsoref) [inline articles/supply-chain/warehousing/step-icons-titles.md:306]: [renamed](https://github.com/MicrosoftDocs/dynamics-365-unified-operations-public/pull/3964/files#diff-6d21d5020d2e616d1f38ce1f6d80149b8a463a344cc58fac2ca88dd9eea214b8)

**Comment by josaw1** [comment]
Thanks for checking the spelling, we’ll take care of these issues in a different process, so I’m closing the PR.

**Pull request closed without merging**